pyiron_atomistics/sphinx/output_parser.py

```diff
@@ -417,3 +418,98 @@ def results(self):
             if len(value) > 0:
                 results["dft"][key] = value
         return results
+
+
+class SphinxWavesParser:
```

Comment:
is "parser" the best name? It actually is a "reader"...

For me, "parsing" sounds as if there is some human-readable text that needs to be processed to extract information. But most of this is plain reading from hdf5, plus a bit of reshaping/reordering.